web/components/organisms/deploy/DeployResolver.tsx

```diff
@@ -138,7 +147,7 @@ export const DeployResolverCard: FC = () => {
             {
                 error && (
                     <p className="text-red-500">
-                        {chainId !== 5 ? 'Only Goerli is supported right now' : error.message}
+                        {(chainId !== 5 || chainId !== 11155111) ? 'Only Goerli and Sepolia is supported right now' : error.message}
```

Comment:
Mainnet is supported too! So maybe this should just be `{error.message}`

Reply:
added all supported chains